seqr/views/utils/pedigree_info_utils_tests.py

```diff
@@ -205,21 +209,21 @@ def test_parse_datstat_pedigree_table(self):
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; __Current Age:__ Patient is deceased, age 33, due to heart attack, sample not available
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; __Age of Onset:__ 21
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; __Race/Ethnicity:__ White, Asian, Pacific; Not Hispanic
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; __Case Description:__ I have a really debilitating probably genetic condition. Ive seen many specialists.
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; __Clinical Diagnoses:__ Yes; SMAs
```

Comment:
we were intentionally stripping unicode here - why did that change? This is a crucial import that does contain unicode and needs to remain stable

Reply:
The change is because of the change in:
```
def _get_datstat_family_notes(row):
-    row = {k: unicode(v, errors='ignore') for k, v in row.items()}
+    row = {k: str(v, 'utf-8', errors='ignore') if not isinstance(v, str) else v for k, v in row.items()}
```
`unicode(v, errors='ignore')` strips unicode chars in `v` when the type of `v` is `str`. But, after converting `pedigree_info_utils_tests.py` to the compatible mode, `v` becomes a `unicode` and `unicode(v, errors='ignore')` cause an exception because `unicode` can't be decoded.
After importing `builtins.str`, `str(unicode_v, errors='ignore')` isn't an exception but it won't strip unicode chars.